src/utils/properties.ts

```diff
@@ -0,0 +1,4 @@
+export const properties = {
+    certificateURL: 'http://localhost:8000/certificates',
```

Comment:
store only base url which is http://localhost:8000. you can append to this url for different end points